### ISSUE TITLE
Red Screen

### ISSUE THREAD

**Issue by rickrmz** [smealum/udsploit#4]
When I'm trying to load the udsploit on the home-brew launcher it gets stuck on a red screen, i've tried several times rebooting the console and trying again but it keeps getting frozen on the screen, what can i do about it?

**Comment by cbonacore** [comment]
What version firmware do you have?

**Comment by sylvain-perez** [comment]
Hi, I have the same issue.
Firmware is 9.9.0-26E

**Comment by cbonacore** [comment]
Did you copy all of the proper files for it?

**Comment by sylvain-perez** [comment]
Holy sh..!!!
I found the problem and it comes from the fact that I use a Mac which has this bad habit of creating extra files...
I was lauching the wrong file "._usdploit.3dsx" which showed up at the top of the list.
If you scroll down you will see the right usdploit with a proper icon instead of the question mark icon ;-)

**Comment by cbonacore** [comment]
Yeah that would explain alot.. in the guide "3ds.guide" it says to scroll all the way down in the Homebrew channel to find it, not on the top.. FYI no offense but you better read the guide word for word or you may brick it. Also I hate apple products lol

**Comment by sylvain-perez** [comment]
Well I carefully read the guide and it says: "You may have to scroll down to see this option".
So one may need to scroll and others no...
Anyway, I managed to launch it, but I am now having issues with safehax which says: PM init failed...
I feel that I will have to spend another couple of hours to figure out how to solve this issue.
However, thanks for your hard work for your exploit.

**Comment by cbonacore** [comment]
If you want I can upload the files I used a few days ago in a part 1 and part 2 folder?

**Comment by sylvain-perez** [comment]
Thank you but I finally found the root cause.
It is coming from the latest build of safehax r21. I used r19 and it works like a charm.
Thanks!

**Comment by cbonacore** [comment]
Ah okay.. let me know if you got anymore problems or questions. Thanks
